packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt

```diff
@@ -50,6 +51,7 @@ class ExpoImageView(context: ReactContext, private val requestManager: RequestMa
   }
   private var loadedSource: GlideUrl? = null
   internal var sourceMap: ReadableMap? = null
+  private var blurRadius: Int? = null
```

Comment:
Let's convert this to actual property with public setter 💪 
```suggestion
  var blurRadius: Int? = null
    private get
```